core/src/main/scala-3/pureconfig/generic/derivation/labels.scala

```diff
@@ -0,0 +1,20 @@
+package pureconfig.generic
+package derivation
+
+import scala.compiletime.{constValue, erasedValue, summonFrom, summonInline}
+import scala.deriving.Mirror
+
+object Labels {
```

Comment:
I'll fix that. It's probably been left with that name because changing just case is a hassle in git, it doesn't register the change on my computer unless I commit the file with another name and then commit it with the correct name and case in the next commit.